app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+end
```

Comment:
- [ ] Kindly add a rails active call back `after_save` to call the `#update_post_counter` method every time a new post has been saved as per requirement. Also, do this for the `update_likes_count` method as well as the `update_comments_count` method in the like model and comment model respectively